apps/web/ui/partners/lander-blocks/ImageBlock.tsx

```diff
@@ -0,0 +1,18 @@
+import { programLanderImageBlockSchema } from "@/lib/zod/schemas/programs";
+import { z } from "zod";
+
+export function ImageBlock({
+  block,
+}: {
+  block: z.infer<typeof programLanderImageBlockSchema>;
+}) {
+  return (
+    <div className="py-4">
+      <img
+        src={block.data.url}
+        alt={block.data.alt}
+        className="block rounded-lg"
+      />
```

Comment:
Should we store image size / aspect ratio data so we can avoid layout shifts when images load?